checks/ad_replication

```diff
@@ -34,29 +34,29 @@
 # showrepl_INFO,Standardname-des-ersten-Standorts,WIN2003,"CN=Schema,CN=Configuration,DC=corp,\
 #DC=de",Standardname-des-ersten-Standorts,WIN2003-DC2,RPC,0,0,2010-07-02 12:46:28,0
 
-ad_replication_default_params = (15, 20)
+# Changelog
+# 2019-06-0 Christian Wirtz doc@snowheaven.de doctore74 - added long output and multiline output
 
+ad_replication_default_params = (15,20)
 
 def parse_ad_replication_dates(s):
     if s == '0' or s == '(never)':
         return None, "unknown"
-
-    s_val = time.mktime(time.strptime(s, '%Y-%m-%d %H:%M:%S'))
-    s_txt = get_relative_date_human_readable(s_val)
-    return s_val, s_txt
-
+    else:
+        s_val = time.mktime(time.strptime(s, '%Y-%m-%d %H:%M:%S'))
+        s_txt = get_relative_date_human_readable(s_val)
+        return s_val, s_txt
 
 def parse_ad_replication_info(info):
     lines = []
     for line_parts in info:
         # Make lines split by , instead of spaces
         line_txt = ' '.join(line_parts).replace(',CN=', ';CN=').replace(',DC=', ';DC=')
         line_parts = line_txt.split(',')
-        if len(line_parts) in [11, 10]:
+        if len(line_parts) in [ 11, 10 ]:
```

Comment:
Use yapf for code formatting with the .style.yapf in the check_mk git root directory instead of manual formatting.